knowledge_verificator/utils/configuration_parser.py

```diff
@@ -0,0 +1,131 @@
+"""Module with the parser for YAML configuration files."""
+
+from dataclasses import dataclass
+from enum import Enum
+from pathlib import Path
+from typing import Any
+import yaml  # type: ignore[import-untyped]
+
+
+class OperatingMode(Enum):
+    """
+    Available mode of operating for the application.
+
+    These modes mean:
+    - EXPERIMENT - implemented experiments on available language models
+        are run. These experiments aim at measuring quality of results
+        produced by the models.
+    - CLI - interactive command-line interface mode.
+        It allows a user to perform actions in terminal.
+    - BACKEND - an HTTP server is started, which serves as backend.
```

Comment:
HTMX confirmed ( ͡° ͜ʖ ͡°) ???

Reply:
Frontend will most likely be written in either `Svelte` or `Vue`. However, any other suggestions are kindly welcomed.